src/main/java/baritone/process/FollowProcess.java

```diff
@@ -50,6 +49,13 @@ public FollowProcess(Baritone baritone) {
 
     @Override
     public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
+        for (Entity entity : ctx.world().loadedEntityList) {
+            if (entity instanceof EntityItem | entity instanceof EntityXPOrb) {
```

Comment:
delicious usage of bitwise or instead of logical or

😋

Reply:
Wait, what is the actual difference? Is there an actual performance effect? Does it sometimes give different results?

Reply:
yes, bitwise or has to check every part, while logical or will shortcut the first time it hit's a "true"

Reply:
Thanks